src/Datadog.NativeLoader/class_factory.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CLASS_FACTORY_H_
+#define CLASS_FACTORY_H_
+
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full
+// license information.
+
+#include <atomic>
+#include "unknwn.h"
+#include "proxy.h"
```

Comment:
you can remove this include and add a forward declaration for `DynamicDispatcher`. This will improve the build.

Reply:
done in 7ff2983